scripts/manage/landing_page_base.md

```diff
@@ -114,6 +114,9 @@ accounts) must be pushed and applied to your deployment to take effect:
 
 ### Add Spinnaker account for GKE
 
+This script grants the Spinnaker deployment's Google Service Account the required
+IAM GKE roles in the project containing the referenced cluster.
```

Comment:
The wording "...in the project containing the reference cluster" is a little different from the others' "...in the managed project." Does it need to be different?

Reply:
Yup. For GCE and GAE, it is specifically the project that we are an adding an account for. For GKE, it is a given cluster (really a k8s context, which can be more narrowly-specified than entire cluster...) which happens to live within a GCP project.

Reply:
:+1: